src/include/miopen/sqlite_db.hpp

```diff
@@ -56,7 +56,7 @@ const auto MIOPEN_SQL_BUSY_TIMEOUT_MS = 60000;
 template <class Derived>
 struct SQLiteSerializable
 {
-    std::vector<std::string> FieldNames() const
+    std::vector<std::string> FieldNames()
```

Comment:
Why is `const` removed? This shouldn't mutate the object.

Reply:
Fixed